Utils/Functions/Net/Download/script.js

```diff
@@ -120,7 +120,7 @@ Downloader.prototype.get = function () {
     }
 
     if (this._localFile) {
-        this._downloader.get(this._url, this._localFile, this._headers, function (progressEntity) {
+        var downloadFile = this._downloader.get(this._url, this._localFile, this._headers, function (progressEntity) {
```

Comment:
What's the difference between `downloadFile` and `_localFile`?

Reply:
_localFile is what you set
downloadFile is what you get

Reply:
I don't understand. Doesn't it mean "please download to `_localFile`"?

Reply:
_localFile can be a directory in fact. So if it is the case, you'll get extra information from the return

Reply:
Shouldn't we make the interface clearer? So either it has to be a file or a directory? I think otherwise it will only be confusing.

Reply:
I don't think so. When you use "cp file destination", destination can be a file or a directory. If it is a directory, the new file's name is destination/file. If it is a file, the new file name is "destination". 
This component behaves exactly the same

Reply:
Ok but this should be documented.

Reply:
Ok but this should be documented.

Reply:
Ok

Reply:
Should be enough if you update the JSDoc for the downloader.